rental.rb

```diff
@@ -0,0 +1,13 @@
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
+
+  def initialize(date, person, book)
```

Comment:
- Fantastic job for breaking the `many-to-many relationship` by setting the Book and Person into this constructor :+1: